Sources/SuperValidator/Validators/Phone Number/PhoneNumberCountry.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  File.swift
+//  
+//
+//  Created by christopher.mienarto on 07/04/22.
+//
+
+import Foundation
+
+public enum PhoneNumberCountry: Equatable {
+    case australia
+    case brazil
+    case india
+    case indonesia
+    case singapore
+    case custom(countryCode: String, nationalPrefix: String?, maxLength: Int?)
+    
+    internal var countryCode: String {
+        switch self {
+        case .australia:
+            return "61"
+        case .brazil:
+            return "55"
+        case .india:
+            return "91"
+        case .indonesia:
+            return "62"
+        case .singapore:
+            return "65"
+        case let .custom(countryCode, _, _):
+            return countryCode
+        }
+    }
+    
+    internal var nationalPrefix: String {
+        switch self {
+        case .australia:
+            return "0"
+        case .brazil:
+            return "0"
+        case .india:
+            return "0"
```

Comment:
If values are same, we can implement a case with a single line
```swift
case .austaralia, .brazil, .India:
 return "0"

Reply:
got it